src/client/locales/default.json

```diff
@@ -313,5 +313,6 @@
   "login_to_busy": "Login to Busy",
   "login_modal_description": "Login with your Steem account using SteemConnect to enjoy Busy at 100%",
   "login_with_steemconnect": "Login with SteemConnect",
-  "follows_you": "Follows you"
+  "follows_you": "Follows you",
+  "transfer_current_usd_amount": "Current USD value of this transfer: ${amount}"
```

Comment:
This isn't used anymore so I think we don't have to touch translations.

Reply:
yeah ill remove it